src/about/AboutScreen.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { View,
+          Text,
+          StyleSheet,
+          TouchableOpacity,
+          Linking,
+        } from 'react-native';
+import Logo from './Logo';
+
+export const BRAND_COLOR = 'rgba(66, 163, 109, 1)';
```

Comment:
You could import styles from '../common/styles' and use BRAND_COLOR from their instead to keep the code DRY